.github/workflows/build.yml

```diff
@@ -26,58 +28,94 @@ jobs:
           mask-aws-account-id: true
           role-to-assume: ${{ secrets.AWS_ROLE_NON_PROD }}
 
-      - name: (NonProd - Imagery Config) Create pull request tag
-        if: github.ref != 'refs/heads/master' && github.event_name == 'pull_request'
+      - name: (NonProd - Config) Bundle Config File
+        run: |
+          docker run -v $PWD:$PWD ${DOCKER_AWS_ENV} ${BASEMAPS_CONTAINER} bundle --config $PWD/config --output $PWD/config.json
+
+      - name: (NonProd - Config) Compare To Production
         run: |
-          ./bin/bmc.js import -t pr-${{ github.event.number }} --commit
+          docker run -v $PWD:$PWD ${DOCKER_AWS_ENV} ${BASEMAPS_CONTAINER} import --config $PWD/config.json
```

Comment:
I think we should compare to `config-latest` ? this means we dont need to log in to production to do the compare

Reply:
This is compare the config.json with the dynamo db data.

Reply:
I know but to do that we are logging into production. I don't think a pull request should log into production.

production should always be the same as config-latest

Reply:
Sorry for the confusion, this is just compare the config files with the non-prod dynamodb configs.